keps/sig-node/2403-pod-resources-allocatable-resources/README.md

```diff
@@ -139,6 +143,7 @@ message ContainerResources {
     string name = 1;
     repeated ContainerDevices devices = 2;
     repeated int64 cpu_ids = 3;
+    repeated ContainerMemory memory = 4;
```

Comment:
just noting: this reflects what is already delivered in code.

Reply:
Yes, that's right. Please refer to the code [here](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/pkg/apis/podresources/v1/api.proto#L53).